src/www/system_certmanager.php

```diff
@@ -832,6 +855,66 @@ function internalca_change() {
                 </tr>
               </tbody>
             </table>
+            <!-- sign_cert_for_csr -->
+            <table id="sign_cert_for_csr" class="table table-striped opnsense_standard_table_form">
+              <thead>
+                <tr>
+                  <th colspan="2"><?=gettext("Sign CSR");?></th>
+                </tr>
+              </thead>
+              <tbody>
+                <tr>
+                  <td style="width:22%"><?=gettext("Certificate authority");?></td>
+                  <td style="width:78%">
+                    <select name='caref_sign_csr' id='caref_sign_csr'>
+  <?php
+                    foreach ($a_ca as $ca) :
+                        if (!$ca['prv']) {
+                            continue;
+                        }?>
+                      <option value="<?=$ca['refid'];?>" <?=isset($pconfig['caref_sign_csr']) && isset($ca['refid']) && $pconfig['caref_sign_csr'] == $ca['refid'] ? 'selected="selected"' : '';?>><?=$ca['descr'];?></option>
+  <?php
+                    endforeach; ?>
+                    </select>
+                    <div class="hidden" id="no_caref_sign_csr">
+                      <?=sprintf(gettext("No internal Certificate Authorities have been defined. You must %sadd%s an internal CA before creating an internal certificate."),'<a href="system_camanager.php?act=new&amp;method=internal">','</a>');?>
+                    </div>
+                  </td>
+                </tr>
+                <tr>
+                  <td style="width:22%"><a id="help_for_csr_sign" href="#" class="showhelp"><i class="fa fa-info-circle"></i></a> <?=gettext("CSR file");?></td>
```

Comment:
The 'CSR file' shows an orange info icon indicatin there's help text available, but it doesn't appear when clicking on it.
Probably due to 'id="help_for_csr_sign"' vs. 'data-for="help_for_csr"' below (line 888)?